src/wallet/rpcwallet.cpp

```diff
@@ -6174,6 +6174,12 @@ UniValue issueasset(const JSONRPCRequest& request)
 
     bool blind_issuances = request.params.size() < 3 || request.params[2].get_bool();
 
+    // Check for optional contract to hash into definition
+    uint256 contract_hash;
```

Comment:
ok to leave unitialized?

Reply:
Yeah the default value is 0. I am assuming C++ calls the default constructor so that this is equivalent to `uint256 contact_hash = uint256();`..

Reply:
Though we usually don't do that though

```
$ rgcpp "uint256 [a-z_]+;" | wc -l
147
$ rgcpp "uint256 [a-z_]+\{\};" | wc -l
6
$ rgcpp "uint256 [a-z_]+ = uint256\(\);" | wc -l
1
```

Reply:
So would it be the same and is it more a principle difference? Or is the uninitiated actually uninitiated and random memory?

Reply:
It is just a principle difference but Core is increasingly moving toward using `{}` everywhere to reduce the cognitive load associated with C++ ambiguity.